src/views/dialogs/DepositDialog2/DepositForm.tsx

```diff
@@ -54,12 +55,13 @@ export const DepositForm = ({
       )}
       <Button
         tw="w-full"
-        state={ButtonState.Disabled}
+        state={isFetching ? ButtonState.Disabled : ButtonState.Default}
         disabled
+        // slotLeft={<LoadingDots />}
         action={ButtonAction.Primary}
         type={ButtonType.Submit}
       >
-        {stringGetter({ key: STRING_KEYS.DEPOSIT_FUNDS })}
+        {isFetching ? <LoadingDots size={3} /> : stringGetter({ key: STRING_KEYS.DEPOSIT_FUNDS })}
```

Comment:
no need to add Loading dots here. I think if you update
```
state={{ isDisabled, isLoading: isFetching }}
```
you should get it for freeeeeeee

Reply:
:O nice!